src/main.zig

```diff
@@ -23,6 +23,10 @@ const Cache = @import("Cache.zig");
 const target_util = @import("target.zig");
 const ThreadPool = @import("ThreadPool.zig");
 const crash_report = @import("crash_report.zig");
+const Module = @import("Module.zig");
+const AstGen = @import("AstGen.zig");
+const Zir = @import("Zir.zig");
+const llvm = @import("codegen/llvm/bindings.zig");
```

Comment:
I don't want these in scope for the rest of the file. Don't want any accidental dependencies on these namespaces.